content/docs/tasks/traffic-management/secure-ingress-sds/index.md

```diff
@@ -0,0 +1,473 @@
+---
+title: Securing Gateways with SDS
+description: Describes how to configure Istio to expose a service outside of the service mesh, over TLS or Mutual TLS, using SDS.
+weight: 31
+keywords: [traffic-management,ingress,sds]
+---
+
+The [Securing Gateways with HTTPS](/docs/tasks/traffic-management/secure-ingress) task describes how to configure an ingress
+gateway to expose an HTTP endpoint of a service to external traffic with either simple or mutual TLS.
+This task shows you how to do the same, but using an ingress gateway agent to provision key/cert and root cert to ingress gateway dynamically.
+
+{{< boilerplate before-you-begin-ingress >}}
+
+## Configure a TLS ingress gateway with secret discovery service enabled
+
+In this section you will configure a TLS ingress gateway that fetches credentials from the ingress gateway agent via secret discovery service (SDS).
+The ingress gateway agent is running in the same pod as the ingress gateway, and watches credentials created in the same namespace as the ingress gateway.
+Enabling SDS at ingress gateway brings the following benefits.
+
+* The ingress gateway is able to dynamically add/delete/update ingress gateway key/certificate and root certificate. You do not have to restart the ingress gateway.
+
+* The secret volume mount is no longer needed. Once you create a `kubernetes` secret, that secret is captured by the gateway agent and sent to ingress gateway as key/certificate or root certificate.
+
+* The gateway agent is able to watch multiple key/certificate pairs. You only need to create secrets for multiple hosts and update the gateway definitions.
+
```

Comment:
Are there any disadvantages to using this approach? If not, should we replace the existing task, instead of having two tasks.

cc @vadimeisenbergibm

Reply:
There are not any disadvantages to using this approach.
We keep the existing task because the file mount based approach is still useful to some customers.
cc @myidpt